tests/stress_tests/test_config.py

```diff
@@ -0,0 +1,56 @@
+log_config = {
```

Comment:
`log_config` should be moved to the tail of config file, because it will be rarely modified

Reply:
log_config should be declared before test_config since test_config refers to log_config